ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bgp_ecmp.schema.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  bgp_ecmp:
+    documentation_options:
+      filename: Fabric Variables
+      table: BGP multi-path Settings
+    description: Maximum ECMP for BGP multi-path
+    type: int
+    convert_types:
+      - str
+    default: 4
```

Comment:
Same question regarding adding ranges for this one and bgp_maximum_paths

(looking at vEOS here)

```
Leaf1(config-router-bgp)#maximum-paths ?
  <1-128>  Value for maximum number of equal cost paths

Leaf1(config-router-bgp)#maximum-paths 42 ?
  ecmp  Maximum number of installed ECMP routes
  <cr>

Leaf1(config-router-bgp)#maximum-paths 42 ecmp ?
  <1-128>  Value for maximum number of installed ECMP routes
```

Reply:
These are very hardware dependent. So be careful using any values from test devices.

Reply:
The 7280R3 can do 512, we can set that as the maximum for now! 
Reference: https://www.arista.com/assets/data/pdf/Datasheets/7800R3-Data-Sheet.pdf

Reply:
addressed in [f94bf27](https://github.com/aristanetworks/ansible-avd/pull/2507/commits/f94bf272c4edf2aca1a925f2d2bf2469136eb45b)